starter-code/routes/auth-routes.js

```diff
@@ -0,0 +1,100 @@
+const express = require("express");
+const authRoutes = express.Router();
+const User = require("../models/user");
+const bcrypt = require("bcrypt");
+const bcryptSalt = 10;
+
+//redirect to signup page
```

Comment:
everything good, except this comment `redirect to singup page` is misleading because the code is not redirecting, it is rendering the page